README.md

```diff
@@ -164,16 +161,16 @@ $ mvn diktat:check
 rule sets by editing the YAML file).
 
 ## Run with Gradle using diktat-gradle-plugin
-Requires a gradle version no lower than 5.3.
+Requires a gradle version no lower than 7.0
 
-This plugin is available since version 0.1.5. You can see how the plugin is configured in our examples: [build.gradle.kts](examples/gradle-kotlin-dsl/build.gradle.kts).
+You can see how the plugin is configured in our examples: [build.gradle.kts](examples/gradle-kotlin-dsl/build.gradle.kts).
```

Comment:
Can you please also add section examples with links to our example projects one by one?
Something like this: `You can see how the plugin is configured in our examples:`
- multi-module project
- groovy dsl
- kotlin dsl

Reply:
updated